docs/userGuide/syntax/panels.mbdf

```diff
@@ -290,7 +290,7 @@ bottom-switch | `Boolean` | `true` | Whether to show an expand switch at the bot
 popup-url | `String` | | The url that the popup window will navigate to. The url can be absolute or relative.
 preload | `Boolean` | `false` | Whether the content is loaded immediately from `src`.
 src | `String` | | The url to the remote page to be loaded as the content of the panel.
-type | `String` | `light` | The type of color for the tab (single).<br>Supports: `light`, `dark`, `primary`, `secondary`, `info`, `success`, `warning`, `danger`, `seamless`.
+type | `String` | `light` | The type or color of the panel (single).<br>Supports: `light`, `dark`, `primary`, `secondary`, `info`, `success`, `warning`, `danger`, `seamless`, `minimal`.
```

Comment:
`color` → `color scheme`? Since we don't support colors directly.

Consistent with Bootstrap's terminology: https://getbootstrap.com/docs/4.3/components/navbar/#color-schemes

Reply:
Noted, I see why it was `type of color` now. Thanks!